docs/source/workflow_outputs.rst

```diff
@@ -93,7 +93,7 @@ Annual energy outputs are listed below.
   ====================================  ===========================
   Type                                  Notes
   ====================================  ===========================
-  Energy Use: Total (MBtu)              Total energy consumption; includes any battery charging/discharging
+  Energy Use: Total (MBtu)              Total energy consumption; includes any home battery charging/discharging, excludes electric vehicle discharging
```

Comment:
Kind of confused by this since it doesn't say anything about EV charging. Is this a simpler version of what you're trying to say?
```suggestion
  Energy Use: Total (MBtu)              Total energy consumption; includes any home battery charging/discharging and EV charging
```

Make similar edits below.